UI/ChatWnd.cpp

```diff
@@ -479,6 +487,18 @@ void MessageWnd::HandlePlayerChatMessage(const std::string& text,
 
     *m_display += wrapped_text + "\n";
     m_display_show_time = GG::GUI::GetGUI()->Ticks();
+
+    // if client empire is target of message, show message window
```

Comment:
What messages are players receiving that aren't targeted to them?

Reply:
E. g. the chat window of the multiplayer lobby will start flashing once a player enters the game.